mlflow/spark.py

```diff
@@ -631,8 +631,8 @@ def predict(self, pandas_df):
 @autologging_integration(FLAVOR_NAME)
 def autolog(disable=False):  # pylint: disable=unused-argument
     """
-    Enables (or disables) and configures logging of Spark datasource paths, versions (if applicable), and formats
-    when they are read. This method is not threadsafe and assumes a
+    Enables (or disables) and configures logging of Spark datasource paths, versions
```

Comment:
Missed this when I approved https://github.com/mlflow/mlflow/pull/3838 via GitHub mobile UI. Lesson learned!


cc @mohamad-arabi

Reply:
Looks like it did: https://github.com/mlflow/mlflow/runs/1553912990. I just didn't catch it while reviewing via a mobile phone web browser :/